src/components/07-form/templates/address-form.njk

```diff
@@ -1,11 +1,19 @@
 <form class="usa-form usa-form--large">
   <fieldset class="usa-fieldset">
     <legend class="usa-legend">Mailing address</legend>
-    <label class="usa-label" for="mailing-address-1">Street address 1</label>
+    <label class="usa-label" for="mailing-address-1">Street Number and Name</label>
     <input class="usa-input" id="mailing-address-1" name="mailing-address-1" type="text">
 
-    <label class="usa-label" for="mailing-address-2">Street address 2 <span class="usa-hint">(optional)</span></label>
-    <input class="usa-input" id="mailing-address-2" name="mailing-address-2" type="text">
+    <label class="usa-label" for="apt-floor-suite">Apartment/Floor/Suite <span class="usa-hint">(optional)</span></label>
+    <select class="usa-select" id="apt-floor-suite" name="apt-floor-suite">
+        <option value>- Select One -</option>
+        <option value="apartment">Apartment</option>
+        <option value="floor">Floor</option>
+        <option value="suite">Suite</option>
+    </select>
+
+    <label class="usa-label" for="apt-floor-suite-no">Unit Number <span class="usa-hint">(optional)</span></label>
+    <input class="usa-input" id="apt-floor-suite-no" name="apt-floor-suite-no" type="text">
 
     <div class="grid-row grid-gap">
       <div class="mobile-lg:grid-col-8">
```

Comment:
Should the following select be a combo box?

Reply:
I don't think the list enough to merit a combo box.

Reply:
I added the code to look exactly like the USPS example.  If we want it a different way then please advise.

Reply:
Is 50+ items not enough for combo box? I'd think a state picker is ideal for combo box

Reply:
The comment on:

> Should the following select be a combo box?

Was pointing to the Floor/Suite/Apartment select. Yeah, a combo box would work.